RSDKv4/ModAPI.cpp

```diff
@@ -507,6 +507,18 @@ void GetModVersion(int *textMenu, int *highlight, uint *id, int *unused)
     AddTextMenuEntry(menu, modList[*id].version.c_str());
 }
 
+void GetModID(int *unused, const char *modName)
+{
+    scriptEng.checkResult = -1;
+
+    for (int n = 0; n < (int)modList.size(); ++n) {
+        if (modList[n].name == modName) {
```

Comment:
This function should check for the mod's `folder` instead of its `name` to be consistent with RSDKv5's ModAPI. The `modName` argument should be renamed to `modID`.